### PR TITLE
Add Python Agent Metadata

### PR DESCRIPTION
**Pull request by TimPansino**
## Give us some context

* Adds metadata for python agent release versions v5.12.0.140 - v5.20.0.149, and v6.4.0.157.
* Requires review from docs engineering team.

### CODE REVIEW

**Review by clarkmcadoo — APPROVED** [review]

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-5121141.mdx [hunk 1]
```diff
@@ -3,6 +3,9 @@ subject: Python agent
 releaseDate: '2020-05-04'
 version: 5.12.1.141
 downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: []
+bugs: ['AIOHTTP http client urls displayed as unknown.url when using YARL URLs', 'Data may not be recorded when tornado's RequestHandler.finish', 'asyncio.ensure_future / asyncio.create_task is unable to spawn new transactions', 'py3: A runtime instrumentation error may have occurred when transactions were finalized', 'Application crash when starting a transaction when one is already active', 'Creating a message transaction with distributed tracing enabled may have resulted in a crash']
```
Comment (clarkmcadoo) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-5121141.mdx:7]: i think we might need to get rid of or escape the `:` here, the frontmatter gets parsed as `yaml`

**Review by nbaenam — DISMISSED** [review]
Thank you for the update! I've fixed an error in the bugs field and added minor modifications.

**Comment by nbaenam** [comment]
Hi @TimPansino, do you need more reviews? Can I merge it?

**Comment by TimPansino** [comment]
@nbaenam bumping it back to @clarkmcadoo for approval before this gets merged.

I took out the colons and cleaned up the wording on the metadata blurbs.

**Comment by clarkmcadoo** [comment]
looks like the build got hung up, going to close/reopen and see if that helps

**Comment by TimPansino** [comment]
Pushed an empty commit to bump the Gatsby run

**Review by nbaenam — APPROVED** [review]

**Pull request merged**